charts/kube-prometheus-stack/README.md

```diff
@@ -217,7 +217,7 @@ You can check out the tickets for this change [here](https://github.com/promethe
 The chart has added 3 [dependencies](#dependencies).
 
 - Node-Exporter, Kube-State-Metrics: These components are loaded as dependencies into the chart, and are relatively simple components
-- Grafana: The Grafana chart is more feature-rich than this chart - it contains a sidecar that is able to load data sources and dashboards from configmaps deployed into the same cluster. For more information check out the [documentation for the chart](https://github.com/helm/charts/tree/master/stable/grafana)
+- Grafana: The Grafana chart is more feature-rich than this chart - it contains a sidecar that is able to load data sources and dashboards from configmaps deployed into the same cluster. For more information check out the [documentation for the chart](https://github.com/grafana/helm-charts/tree/main/charts/grafana)
```

Comment:
Per #25 this should have been one sentence per line, I believe even in a list. Would you mind fixing this while you're here?

Reply:
on it.

Reply:
@scottrigby added this and pushed, how does it looks like?

```
- Grafana: The Grafana chart is more feature-rich that contains a sidecar that is able to load data sources and dashboards from configmaps deployed into the same cluster and more information on that can be found by visiting the [documentation for the chart](https://github.com/grafana/helm-charts/tree/main/charts/grafana)
```

Reply:
I think I may be wrong about needing one sentence per line in a list. I'm not on my laptop now but from a quick GitHub flavored markdown test in a comment shows the line break is rendered if within a list... so maybe just keep as you had it.

Test:
- one sentence. Next sentence same line
- one sentence.
Next sentence on a new line

Reply:
@scottrigby 

there’s an issue even with the new word counts all over.
The default statement length seems to be “80” words, not sure we want to go with that count across the community.

This is the error with my new statement
```
README.md:220:81 MD013/line-length Line length [Expected: 80; Actual: 322]
```


for the original, it also errors about the new line as well.
```
README.md:219 sentences-per-line Each sentence should be on its own line [Context: "cluster. For m"]
README.md:220:81 MD013/line-length Line length [Expected: 80; Actual: 310]
```

So I am reverting to the old statement as of now.

I guess we have to agree unanimously what’s the statement’s word count should be on a single sentence. Then tackle on this issue. Thoughts ?

Reply:
I think the original statement you had is great 👍 And yes, probably best to follow up about these questions on #25. Thank you!